### ISSUE TITLE
Storybook UI has a11y violations

### ISSUE THREAD

**Issue by hellobrian** [storybooks/storybook#1520]
## Problem

When using [aXe chrome extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) or the [storybook-addon-a11y](https://github.com/storybooks/storybook-addon-a11y), there are a few a11y violations that can be resolved with storybook's UI. 

Basically, these violations make it a little troublesome to pinpoint whether a developer's components are causing a11y violations or if storybook is causing a11y violations.

## Example Screenshot

![image](https://user-images.githubusercontent.com/4185382/28582721-1136e920-712c-11e7-86a0-20cec9d9e11d.png)

## Todo

- [ ] Elements must have sufficient color contrast
- [ ] `<html>` element must have a lang attribute
- [ ] Form elements must have labels

**Comment by lokywin** [comment]
Hi @hellobrian since you mentioned `storybook-addon-a11y` I have been trying to use it but so far unsuccessful to get it run. Do you have any tips for enabling it? I followed the instructions here (https://github.com/storybooks/storybook-addon-a11y) I am on storybook v2.35.3.

Thanks for any tips and sorry for hijacking this thread.
![28953067-77d1a692-788a-11e7-8cc1-0d230681a304](https://user-images.githubusercontent.com/545713/28981156-7b602222-7905-11e7-8047-48e4789b2698.png)

**Comment by hellobrian** [comment]
Yeah, here's our React component library where we're using it: 
- [.storybook/addons.js](https://github.com/carbon-design-system/carbon-components-react/blob/master/.storybook/addons.js#L3)
- [.storybook/config.js](https://github.com/carbon-design-system/carbon-components-react/blob/master/.storybook/config.js#L4-L9)

Hope this helps!

**Comment by lokywin** [comment]
Thank you very much @hellobrian for a prompt response. You replied quicker than I could type. BTW, I ❤️  you and your team's work on Carbon. The fact that you guys also care for a11y is 🎉  🙇
